src/components/Creepypasta.js

```diff
@@ -1,37 +1,57 @@
 import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import arrow from '../Images/Arrow.svg';
 
 export default function Creepypasta({ title, image, text, wordcount }) {
   return (
     <Wrapper>
-      <Title>{title}</Title>
+      <GoBackTop to="/">
+        <img
+          alt="Red Arrow, if clicked leads back to main page"
+          src={arrow}
+          width="60"
+        />
+      </GoBackTop>
       <Image alt="" src={image} height="160" />
```

Comment:
You could give the picture the title of the story as an alt tag to improve accessibility.